### ISSUE TITLE
Homepage georeferencer text is a bit wordy

### ISSUE THREAD

**Issue by edtjones** [layersoflondon/application#233]
Might benefit from being truncated a bit @selrashidi 

<img width="765" alt="screen shot 2018-08-03 at 21 10 11" src="https://user-images.githubusercontent.com/4377783/43663512-b0b4cbbe-9761-11e8-8c31-abd77557bd36.png">

**Comment by amytodd** [comment]
@selrashidi it is on the homepage - feel free to change. I changed it temporarily so as not to advertise the geo-referencing but to highlight the school sign up for this but I can shorten it?

**Comment by andyError** [comment]
No longer an issue.
